targets/ChibiOS/RP_PICO_W_RP2040/target_littlefs.c

```
@@ -0,0 +1,142 @@
+// Reference to ChibiOS EFL driver instance
+extern EFlashDriver EFLD1;
+
+// Buffers required by littlefs_FS_Driver
+uint8_t lfs_inputBuffer[RP2040_FLASH_PAGE_SIZE];
+uint8_t lfs_outputBuffer[RP2040_FLASH_PAGE_SIZE];
+
+int32_t lfs_inputBufferSize = RP2040_FLASH_PAGE_SIZE;
+int32_t lfs_outputBufferSize = RP2040_FLASH_PAGE_SIZE;
+
+// Convert a littlefs block+offset to a flash offset relative to XIP base
+static inline flash_offset_t lfs_to_flash_offset(const struct lfs_config *c, lfs_block_t block, lfs_off_t off)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'lfs_to_flash_offset' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)